src/getPostcssResult.js

```diff
@@ -43,13 +43,21 @@ export default function (
       } else if (syntax === "sugarss" || !syntax && fileExtension === ".sss") {
```

Comment:
Have you considered the PostCSS plugin use-case — where somebody inserts stylelint as a PostCSS plugin, using neither the Node API nor the CLI?